hermes/filters.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, data=None, queryset=None, *, request=None, prefix=None):
         # not possible in class variable assignment of MultipleChoiceFilter.
         # see https://github.com/carltongibson/django-filter/blob/main/django_filters/filterset.py#L309
         # see https://github.com/carltongibson/django-filter/blob/main/django_filters/fields.py#L253
-        self.get_filters()['topic'].field.choices = [(t, t) for t in get_all_public_topics()]
+        self.filters['topic'].extra['choices'] = [(t, t) for t in get_all_public_topics()]
```

Comment:
This was a to fix a bug where the first time after the server reloads from a change (locally at least), the topic filter would fail because it would not be initialized with any options yet. Doing it this way works the first time.